content/html/concepts/semantic-html/terms/details/details.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '<details>'
+Description: 'Encapsulated the <summary> element and any additional details which are only visible when the element is in an open state.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<details>` semantic HTML element encapsulated the <summary> element and any additional elements which are only visible when the element is in an open state.
+There is a small triangle which shows the state of the element and by clicking on the element or using the space bar you can toggle between revealing and hiding additional information.
```

Comment:
I think this sentence could be broken into two smaller ones. Maybe the text starting with "by clicking on the element..." could be the beginning of the second one. Also, per the content standards, let's avoid first- and second-person pronouns like "we" and "you", if possible, since this content is just a reference entry.